### PR TITLE
[WEBSITE] Add FAQ item for Forseti vs. SHA

### PR DESCRIPTION
**Pull request by hshin-g**
Addresses #3295.

Below is a screenshot of what this looks like:

Screen Shot 2019-11-07 at 10 10 07 AM

Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [x] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [x] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [x] My PR has been functionally tested.
- [x] All of the [tests](https://forsetisecurity.org/docs/latest/develop/dev/testing.html) pass
- [x] I have submitted a corresponding PR in the [Forseti Terraform module](https://github.com/forseti-security/terraform-google-forseti) (if applicable).

These guidelines and more can be found in our [contributing guidelines](https://github.com/forseti-security/forseti-security/blob/dev/.github/CONTRIBUTING.md).

### CODE REVIEW

File: _faq/general/forseti-sha.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+* If you want to be hands on, Forseti is a great solution to integrate into new or existing tooling.
+* If you want to be more hands free, Security Health Analytics is a great solution for a managed service.
+
+{: .table}
+|Forseti|Security Health Analytics|
+|--------|------------|
+|Customer deployed and managed|Fully managed by GCP with SLA|
+|Community support|GCP support|
+|Customizable auditing for policy-as-code ecosystem|Comprehensive set of benchmarks (e.g. CIS certified)|
```
[the remark this answers is not in the document]
Comment (blueandgold) [reply _faq/general/forseti-sha.md]: Would also describe `policy-as-code` a bit more.  Please see the description in the original issue.

[the remark this answers is not in the document]
Comment (hshin-g) [reply _faq/general/forseti-sha.md]: Thanks @blueandgold - have updated with comments. Please take a look!

**Comment by hshin-g** [comment]
I have updated the screenshot above to reflect what the docs would look
like with these changes- let me know what you think!

**Comment by blueandgold** [comment]
The updated screenshot looks great.  Though to nitpick, the left column is a bit wide because of the long line.  Is a way to wrap the line (possibly with `<br>` tag)?

**Comment by hshin-g** [comment]
@blueandgold Thanks for the tip- have added in the tag and updated the screenshot with what it looks like now. Lmk what you think!

**Comment by blueandgold** [comment]
Perfect!

**Pull request merged**